.github/workflows/smoke-test.yaml

```
@@ -47,7 +47,7 @@ jobs:
       - name: Run helm-docs
         run: |
           make -C install/kubernetes docs
-          git diff --exit-code
+          git diff --exit-code || (echo "please run 'make -C install/kubernetes docs' and check in files"; exit 1)
```

Nit: all other messages say "...and submit your changes" here. Maybe change it as well to stay consistent?
```suggestion
          git diff --exit-code || (echo "please run 'make -C install/kubernetes docs' and submit your changes"; exit 1)
```

---

good pick, thanks :100: